examples_tests/ComputePointCloud/rasterizer.comp

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#version 450
+
+#pragma shader_stage(compute)
+
+layout (local_size_x = 256, local_size_y = 1) in;
+
+layout (push_constant) uniform pushConstants
+{
+	uvec2 imgSize;
+	uint pointCount;
+	mat4 mvp;
+} u_pushConstants;
+
+layout (set = 0, binding = 0, r32ui) uniform coherent uimage2D outImage;
+layout (set = 0, binding = 1, std430) readonly buffer pointCloud {
+	vec3 position[];
+} u_pointCloud;
+
+void main()
+{
+	if (gl_GlobalInvocationID.x >= u_pushConstants.pointCount) 
+	{
+		return;
+	}
```

Comment:
need to use "virtual" (persistently resident) workgroups otherwise your rasterizer wont handle more than 256*64k = 16M points on AMD GPUs

Reply:
you basically want to launch a fixed number of workgroups (see `CScanner` or any other built-in compute thing) which run on a loop and comupute a `virtualInvocationIndex` instead of using `gl_GlobalInvocationID.x`